src/components/Municipality/MunicipalityProgress/index.js

```diff
@@ -42,17 +42,36 @@ export const MunicipalityProgress = ({
           startDate={false}
         />
         <h3>{getSignupsLabel(municipality.signups, municipality.goal)}</h3>
-        {showDescription && (
-          <p>
-            Wenn wir ingesamt {municipality.goal} Menschen sind, dann trauen wir
-            uns zu, gemeinsam den offiziellen Bürgerbegehrensprozess für
-            unseren Modellversuch zu starten. Hierfür müssen wir möglichst viele
-            sein. Hol also noch Menschen dazu!
-          </p>
-        )}
+        {/* Berlin Hamburg Bremen */}
+        {showDescription &&
+          (municipality.ags === '11000000' ||
+            municipality.ags === '02000000' ||
+            municipality.ags === '04011000') && (
+            <p>
+              Um den offiziellen Volksbegehrensprozess in die zweite,
+              entscheidende Runde zu bringen, brauchen wir mindestens{' '}
+              {municipality.goal.toLocaleString('de-DE')} Menschen aus{' '}
+              {municipality.name} – und am besten so viele wie möglich! Hol also
+              noch Menschen dazu.
+            </p>
+          )}
+        {/* all other municipalities */}
+        {showDescription &&
+          !(
+            municipality.ags === '11000000' ||
+            municipality.ags === '02000000' ||
+            municipality.ags === '04011000'
+          ) && (
+            <p>
+              Um den offiziellen Bürgerbegehrensprozess zu starten, brauchen wir
+              mindestens {municipality.goal.toLocaleString('de-DE')} Menschen
+              aus {municipality.name} – und am besten so viele wie möglich! Hol
+              also noch Menschen dazu.
+            </p>
+          )}
       </div>
     );
-  } else if (municipality) {
+  } else if (statsReady) {
```

Comment:
Changed the condition here, because there’s the possibility that the goal hasn’t been fetched in a render cycle yet and the .toLocaleSting method would cause a bug there.

Reply:
ok!